si-tests/_support/index.js

```diff
@@ -0,0 +1,179 @@
+    const groupLabel = Cypress.$(group).find('label');
+    if (groupLabel.length === 0) {
+      return false;
+    }
+
+    return groupLabel[0].innerText.trim().toLowerCase() === compareLabel;
+  });
+
+  // If nothing found, return empty selection
+  expect(formGroup).not.to.equal(null);
+
+  // If we found a form group, return the input element within
+  return formGroup.find('textarea, input, select, .form-control');
+});
+
+/**
+ * Return the value of the configuration row with the given name
+ */
+Cypress.addChildCommand('configurationMapValue', function (elements, label) {
+  const foundElements = [];
+  const compareLabel = label.toLowerCase();
+
+  elements.each(function (index, element) {
+    const labelElement = element.querySelector('.configuration-map-label');
+    if (!labelElement) {
+      return;
+    }
+
+    if (labelElement.innerText.trim().toLowerCase() === compareLabel) {
+      const valueElement = element.querySelector('.configuration-map-value');
+      expect(valueElement).not.to.equal(null);
+      foundElements.push(valueElement);
+    }
+  });
+
+  return Cypress.$(foundElements);
+});
+
+/**
+ * Locate a table row that contains the given string and return the entire row
+ */
+Cypress.addChildCommand('getTableRowThatContains', function (elements, tdContains) {
+  let matchedRows = elements.find('tr');
+
+  matchedRows = matchedRows.filter(function (index, row) {
+    return Array.prototype.slice.apply(row.childNodes).some(function (child) {
+      return child.innerText.indexOf(tdContains) !== -1;
+    });
+  });
+
+  return Cypress.$(matchedRows);
+});
+
+/**
+ * Return all the <td> cells for the given column of the selected table
+ */
+Cypress.addChildCommand('getTableColumn', function (elements, columNameOrIndex) {
+  const matchedRows = elements.find('tr');
+  const headings = matchedRows.eq(0).find('th');
+  let columnIndex = parseInt(columNameOrIndex);
+
+  if (Number.isNaN(columnIndex)) {
+    const compareName = String(columNameOrIndex).toLowerCase();
+
+    // Fallback to first column
+    columnIndex = 0;
+    headings.each(function (index, th) {
+      if (th.innerText.trim().toLowerCase() === compareName) {
+        columnIndex = index;
+      }
+    });
+  }
+
+  // Collect all visible rows
+  return matchedRows
+    .slice(1)
+    .filter(function (index, element) {
+      return Cypress.$(element).is(':visible');
+    })
+    .map(function (index, element) {
+      return element.querySelectorAll('td')[columnIndex];
+    });
+});
+
+/**
+ * Return the text contents of all the selected elements
+ */
+Cypress.addChildCommand('contents', function (elements) {
+  return elements.map((index, element) => element.innerText).get();
+});
+
+/**
+ * Extract the string from the given element (including ACE editor) and
+ * check if the parsed JSON equals to the given JSON object.
+ */
+Cypress.addChildCommand('shouldJsonMatch', function (elements, json) {
```

Comment:
Well, it's a `.should` command, and it operates on various elements (including the ACE editor).

Another idea would be to split this in `.aceContents().asJSONObject()`